trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapCoverEndDate.api-test.js

```diff
@@ -50,4 +50,20 @@ describe('mapCoverEndDate', () => {
       expect(yearResult).toEqual('2021');
     });
   });
+
+  describe('when cover end date is invalid', () => {
+    it('should return 4 digit year', () => {
```

Comment:
it's returning undefined, not a 4 digit year?

Reply:
done